### PR TITLE
GameDB: add HW fixes for snowblind engine games

### PR DESCRIPTION
**Pull request by Mrlinkwii**
add HW fixes for snowblind engine games
### Suggested Testing Steps

Make sure CI is happy

### CODE REVIEW

File: bin/resources/GameIndex.yaml [hunk 1]
```diff
@@ -13314,6 +13330,8 @@ SLES-52388:
 SLES-52392:
   name: "Combat Elite - WWII Paratroopers [Preview]"
   region: "PAL-E"
+  gsHwFixes:
+    texturePreloading: 1 # Performs much better with partial preload
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:13334]: period

File: bin/resources/GameIndex.yaml [hunk 2]
```diff
@@ -39589,6 +39636,8 @@ SLUS-20714:
 SLUS-20715:
   name: "Combat Elite - WWII Paratroopers"
   region: "NTSC-U"
+  gsHwFixes:
+    texturePreloading: 1 # Performs much better with partial preload
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:39640]: period

File: bin/resources/GameIndex.yaml [hunk 3]
```diff
@@ -40321,6 +40372,8 @@ SLUS-20888:
   name: "Front Mission 4"
   region: "NTSC-U"
   compat: 5
+  gameFixes: 
+    - SoftwareRendererFMVHack # Fixes FMVs
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:40376]: period

File: bin/resources/GameIndex.yaml [hunk 4]
```diff
@@ -45443,6 +45500,8 @@ SLUS-28053:
 SLUS-28054:
   name: "Combat Elite - WWII Paratroopers [Trade Demo]"
   region: "NTSC-U"
+  gsHwFixes:
+    texturePreloading: 1 # Performs much better with partial preload
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:45504]: period

File: bin/resources/GameIndex.yaml [hunk 5]
```diff
@@ -45818,6 +45879,8 @@ SLUS-29126:
   name: "Champions - Return to Arms [Demo]" # aka "Champions of Norrath 2 [Demo]"
   region: "NTSC-U"
   compat: 4
+  gsHwFixes:
+    texturePreloading: 1 # Performs much better with partial preload
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:45883]: period

File: bin/resources/GameIndex.yaml [hunk 6]
```diff
@@ -9850,6 +9854,8 @@ SLES-50672:
   name: "Baldur's Gate - Dark Alliance"
   region: "PAL-M5"
   compat: 5
+  gsHwFixes:
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:9857]: Needs capital W

File: bin/resources/GameIndex.yaml [hunk 7]
```diff
@@ -11688,9 +11694,13 @@ SLES-51523:
 SLES-51525:
   name: "Fallout - Brotherhood of Steel"
   region: "PAL-M3"
+  gsHwFixes:
+    texturePreloading: 1 # Performs much better with partial preload.
 SLES-51526:
   name: "Fallout - Brotherhood of Steel"
   region: "PAL-M3"
+  gsHwFixes:
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:11702]: Needs capital W

File: bin/resources/GameIndex.yaml [hunk 8]
```diff
@@ -11688,9 +11694,13 @@ SLES-51523:
 SLES-51525:
   name: "Fallout - Brotherhood of Steel"
   region: "PAL-M3"
+  gsHwFixes:
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:11697]: Needs capital W

File: bin/resources/GameIndex.yaml [hunk 9]
```diff
@@ -13181,6 +13193,8 @@ SLES-52323:
 SLES-52325:
   name: "Champions of Norrath - Realms of EverQuest"
   region: "PAL-M3"
+  gsHwFixes:
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:13196]: Needs capital W

File: bin/resources/GameIndex.yaml [hunk 10]
```diff
@@ -13267,6 +13281,8 @@ SLES-52372:
 SLES-52373:
   name: "Champions of Norrath"
   region: "PAL-E-S"
+  gsHwFixes:
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:13284]: Needs capital W

File: bin/resources/GameIndex.yaml [hunk 11]
```diff
@@ -13315,6 +13331,8 @@ SLES-52388:
 SLES-52392:
   name: "Combat Elite - WWII Paratroopers [Preview]"
   region: "PAL-E"
+  gsHwFixes:
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:13334]: Needs capital W

File: bin/resources/GameIndex.yaml [hunk 12]
```diff
@@ -14417,6 +14435,8 @@ SLES-52889:
 SLES-52894:
   name: "Bard's Tale, The"
   region: "PAL-E"
+  gsHwFixes:
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:14438]: Needs capital W

File: bin/resources/GameIndex.yaml [hunk 13]
```diff
@@ -14849,6 +14869,8 @@ SLES-53039:
   name: "Champions - Return to Arms" # aka "Champions of Norrath 2"
   region: "PAL-M4"
   compat: 5
+  gsHwFixes:
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:14872]: Needs capital W

File: bin/resources/GameIndex.yaml [hunk 14]
```diff
@@ -15096,6 +15118,8 @@ SLES-53152:
 SLES-53154:
   name: "Bard's Tale, The"
   region: "PAL-M8"
+  gsHwFixes:
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:15121]: Needs capital W

File: bin/resources/GameIndex.yaml [hunk 15]
```diff
@@ -17990,6 +18014,8 @@ SLES-54421:
 SLES-54423:
   name: "Justice League Heroes"
   region: "PAL-M5"
+  gsHwFixes:
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:18017]: Needs capital W

File: bin/resources/GameIndex.yaml [hunk 16]
```diff
@@ -21479,6 +21505,8 @@ SLKA-25244:
 SLKA-25246:
   name: "Bard's Tale, The"
   region: "NTSC-K"
+  gsHwFixes:
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:21508]: Needs capital W

File: bin/resources/GameIndex.yaml [hunk 17]
```diff
@@ -27425,6 +27455,8 @@ SLPM-65914:
 SLPM-65915:
   name: "Fallout - Brotherhood of Steel"
   region: "NTSC-J"
+  gsHwFixes:
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:27458]: Needs capital W

File: bin/resources/GameIndex.yaml [hunk 18]
```diff
@@ -33225,6 +33257,8 @@ SLPS-25139:
   name: "Baldur's Gate - Dark Alliance"
   region: "NTSC-J"
   compat: 5
+  gsHwFixes:
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:33260]: Needs capital W

File: bin/resources/GameIndex.yaml [hunk 19]
```diff
@@ -33680,6 +33714,8 @@ SLPS-25291:
   name: "Baldur's Gate - Dark Alliance [PCCW The Best]"
   region: "NTSC-J"
   compat: 5
+  gsHwFixes:
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:33717]: Needs capital W

File: bin/resources/GameIndex.yaml [hunk 20]
```diff
@@ -36763,6 +36799,10 @@ SLUS-20035:
   name: "Baldur's Gate - Dark Alliance"
   region: "NTSC-U"
   compat: 5
+  gameFixes:
+    - SoftwareRendererFMVHack # Fixes fmvs
+  gsHwFixes:
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:36804]: Needs capital W

File: bin/resources/GameIndex.yaml [hunk 21]
```diff
@@ -38820,6 +38860,8 @@ SLUS-20539:
   name: "Fallout - Brotherhood of Steel"
   region: "NTSC-U"
   compat: 5
+  gsHwFixes:
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:38863]: Needs capital W

File: bin/resources/GameIndex.yaml [hunk 22]
```diff
@@ -38950,6 +38992,8 @@ SLUS-20565:
   name: "Champions of Norrath"
   region: "NTSC-U"
   compat: 5
+  gsHwFixes:
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:38995]: Needs capital W

File: bin/resources/GameIndex.yaml [hunk 23]
```diff
@@ -39593,6 +39640,8 @@ SLUS-20714:
 SLUS-20715:
   name: "Combat Elite - WWII Paratroopers"
   region: "NTSC-U"
+  gsHwFixes:
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:39643]: Needs capital W

File: bin/resources/GameIndex.yaml [hunk 24]
```diff
@@ -39976,6 +40025,8 @@ SLUS-20803:
   name: "Bard's Tale, The"
   region: "NTSC-U"
   compat: 5
+  gsHwFixes:
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:40028]: Needs capital W

File: bin/resources/GameIndex.yaml [hunk 25]
```diff
@@ -40744,6 +40797,8 @@ SLUS-20973:
   name: "Champions - Return to Arms" # aka "Champions of Norrath 2"
   region: "NTSC-U"
   compat: 5
+  gsHwFixes:
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:40800]: Needs capital W

File: bin/resources/GameIndex.yaml [hunk 26]
```diff
@@ -45447,6 +45504,8 @@ SLUS-28053:
 SLUS-28054:
   name: "Combat Elite - WWII Paratroopers [Trade Demo]"
   region: "NTSC-U"
+  gsHwFixes:
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:45507]: Needs capital W

File: bin/resources/GameIndex.yaml [hunk 27]
```diff
@@ -45734,6 +45793,8 @@ SLUS-29087:
 SLUS-29088:
   name: "Champions of Norrath - Realms of EverQuest [Demo]"
   region: "NTSC-U"
+  gsHwFixes:
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:45796]: Needs capital W

File: bin/resources/GameIndex.yaml [hunk 28]
```diff
@@ -45822,6 +45883,8 @@ SLUS-29126:
   name: "Champions - Return to Arms [Demo]" # aka "Champions of Norrath 2 [Demo]"
   region: "NTSC-U"
   compat: 4
+  gsHwFixes:
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:45886]: Needs capital W

File: bin/resources/GameIndex.yaml [hunk 29]
```diff
@@ -36763,6 +36799,10 @@ SLUS-20035:
   name: "Baldur's Gate - Dark Alliance"
   region: "NTSC-U"
   compat: 5
+  gameFixes:
+    - SoftwareRendererFMVHack # Fixes fmvs
```
Comment (RedDevilus) [inline bin/resources/GameIndex.yaml:36803]:
```suggestion
    - SoftwareRendererFMVHack # Fixes FMVs.
```

**Pull request merged**